### PR TITLE
Feat/tiles only for tweets [Delivers #116158339]

### PR DESCRIPTION
**Pull request by chiefGui**
@chute/frontend

### CODE REVIEW

File: src/components/live-displays/Grid/GridAsset.scss [hunk 1]
```diff
@@ -160,9 +161,6 @@ $column2-left-position: $asset--normal-width * 2;
   color: rgb(31, 31, 31);
   font-style: oblique;
   font-size: 18px;
-  -webkit-line-clamp: 7;
-  -webkit-box-orient: vertical;
-  text-overflow: ellipsis;
```
Comment (mareksuscak) [inline src/components/live-displays/Grid/GridAsset.scss]: Was this removed intentionally? Just curious.

Comment (chiefGui) [inline src/components/live-displays/Grid/GridAsset.scss]: > Removing ellipsis because the tile is designed for 140 characters max

Comment (mareksuscak) [inline src/components/live-displays/Grid/GridAsset.scss]: :+1:

**Comment by mareksuscak** [comment]
:+1:

**Pull request merged**